packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx

```diff
@@ -0,0 +1,179 @@
+  const [arbiterName, setArbiterName] = useState(arbiterInfo?.userName ?? '');
+
+  const inputBoxesContent: [
+    string,
+    React.Dispatch<React.SetStateAction<string>>,
+    string,
+  ][] = [
+    [pgnUrl, setpgnUrl, 'Tournament Url'],
+    [arbiterPin, setArbiterPin, 'Arbiter Pin'],
+    [arbiterEmailSecret, setArbiterEmailSecret, 'Arbiter Email Secret'],
+    [divisionId, setdivisionId, 'Division Id'],
+    [arbiterId, setArbiterId, 'Arbiter Id'],
+    [arbiterName, setArbiterName, 'Arbiter Name'],
+    [eventName, seteventName, 'Event Name'],
+  ];
+
+  const validateArbiterInfo = (arbiterInfo: ArbiterInfo): boolean => {
+    if (arbiterInfo.arbiterToken == '') {
+      showError('Arbiter Email token cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.divisionId == '') {
+      showError('Division Id cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.eventName == '') {
+      showError('Event Name cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.userName == '') {
+      showError('Arbiter Name cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.userId == '') {
+      showError('Arbiter Id cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.pin == '') {
+      showError('Arbiter Pin cannot be blank');
+      return false;
+    }
+    if (!validUrl(arbiterInfo.broadcastUrl)) {
+      showError('Invalid Pgn Url');
+      return false;
+    }
+    return true;
+  };
+
+  const confirmArbiterInfo = async (): Promise<void> => {
+    const newArbiterInfo: ArbiterInfo = {
+      pin: arbiterPin.trim().replace(/^\s+|\s+$/g, ''),
+      broadcastUrl: pgnUrl.trim().replace(/^\s+|\s+$/g, ''),
+      arbiterToken: arbiterEmailSecret.trim().replace(/^\s+|\s+$/g, ''),
+      divisionId: divisionId.trim().replace(/^\s+|\s+$/g, ''),
+      userId: arbiterId.trim().replace(/^\s+|\s+$/g, ''),
+      userName: arbiterName.trim().replace(/^\s+|\s+$/g, ''),
+      eventName: eventName.trim().replace(/^\s+|\s+$/g, ''),
+    };
+    dismiss();
+
+    if (validateArbiterInfo(newArbiterInfo)) {
+      await setArbiterInfo(newArbiterInfo);
+    }
+  };
+  return (
+    <>
+      <Sheet title="Edit Event" dismiss={dismiss} visible={display}>
+        <View
+          style={{ display: 'flex', flexDirection: 'row', paddingBottom: 20 }}>
+          <IconButton
+            size={60}
+            icon={inScanQRMode ? 'title' : 'qr-code-scanner'}
+            onPress={() => setInScanQRMode(!inScanQRMode)}
+            colour="black"
+            style={styles.switchInputModeButton}
+          />
+          <PrimaryText
+            size={26}
+            weight={FontWeight.Medium}
+            style={styles.instructions}>
+            Go to{' '}
+            <Link
+              style={styles.instructionsLink}
+              label="tornelo.com"
+              link={torneloUrl}
+            />{' '}
+            and either generate a QR code or manually edit the fields here
```

Comment:
It isn't clear how to actually manually edit the fields, I had to watch the video on the other pr to see how to do it

Reply:
yeh i was worried about that, I changed it to look like this: 

![image](https://user-images.githubusercontent.com/26475724/195967212-2f718a4a-ec31-4374-9cc2-498c0390598e.png)

![image](https://user-images.githubusercontent.com/26475724/195967216-0d062e7a-6f75-4536-b03a-b57458e78ea9.png)

Is that better?